src/snek5000/util/__init__.py

```diff
@@ -112,7 +112,7 @@ def activate_paths():
     """Setup environment variables in preparation for Nek5000 build.
```

Comment:
We can remove this function.

Reply:
Done in fad6e499072ea3e69223554921dc611f4590866d